docling_core/types/doc/document.py

```diff
@@ -1106,7 +1106,7 @@ def export_to_dict(self) -> Dict:
         """export_to_dict."""
         return self.model_dump(mode="json", by_alias=True, exclude_none=True)
 
-    def export_to_markdown(  # noqa: C901
```

Comment:
I wouldn't rename `export_to_markdown` to `export_to_markdown_v1`, to not break compatibility

Reply:
it does not, we have export_to_markdown which points by default to v1